docs/whats-new.md

```diff
@@ -44,13 +44,17 @@ If you prefer sharing your files through another app:
 If you have questions or need additional support, don't hesitate to reach out at contact@tella-app.org
 
 
-### iOS: Tella 1.8.0 (70) - Released on June 3, 2024 {#ios-tella-180-70---released-on-june-3-2024}
--  Edit media: Users can crop and rotate images within Tella's encrypted container. [Read more](/features#edit-media).
-- Bug fixes:
-  - Placeholder with current name missing from rename file while recording.
-  - Rename folder shows the label "Rename" instead of "Rename File".
-  - Show attempts selected on the "Delete after failed unlock" and a handle longer languages.
-  - Solved UI issue in the label when creating PIN when installing Tella (only in iOS 14).
+### iOS: Tella 1.9.0 (74) - Released on June 25, 2024 {#ios-tella-190-74---released-on-june-25-2024}
+- Added support for the relationship property type in the Uwazi connection.
+- Added offline support for the Uwazi server connection (Drafts, Outbox and Submitted tabs). More details [here](/uwazi).
+-  Bug fixing and small improvements:
+  - Run all remaining database migrations on every update. This bug was causing an inconsistent database state for users who don't use Tella often.
+  - Fix the strip EXIF metadata feature that had been broken since the last database migration.
+  - Show a warning if the users tries to delete a file that's being used in server connection that's not submitted.
+  - Hide password field in a new Tella Web server connection.
+  - Make the select field on Uwazi connections scrollable.
+  - Fix back button issues on the onboarding flow.
+  - Fix the import functionality when no permissions has been granted.
 
```

Comment:
maybe we can also add :
`The user can select multiple files from the Photo Library.`

Reply:
nice! for import you mean? Is it there a task for that in asana that  I can refer to?
If there isn't, tell me and also let's bring it to co-design just to make sure Figma is update and android has a similar task as well

Reply:
cc @dhekra-rouatbi

Reply:
@carohadad I haven't created a task for that but it was fixed by the Exiff feature. Yes we can discuss that in the co-design

Reply:
ok then ! I won't put it in the release notes, so it's a hidden feature for now. just top of mind, we need to double check how the "delete original" feature should work (meaning, delete all!) and maybe update the text for the confirmation box. And make sure android behaves similrarly